### ISSUE TITLE
x/sys/unix: does not build under 1.4 any more

### ISSUE THREAD

**Issue by davecheney** [golang/go#10854]
arm64 and ppc64 are not know as GOARCHes to Go 1.4
```
% $HOME/go1.4/bin/go build 
# golang.org/x/sys/unix
./syscall_linux_arm64.go:65: Getpagesize redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:73
./syscall_linux_arm64.go:70: TimespecToNsec redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:87
./syscall_linux_arm64.go:72: NsecToTimespec redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:89
./syscall_linux_arm64.go:78: TimevalToNsec redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:95
./syscall_linux_arm64.go:80: NsecToTimeval redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:97
./syscall_linux_arm64.go:87: Pipe redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:106
./syscall_linux_arm64.go:100: Pipe2 redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:119
./syscall_linux_arm64.go:111: (*PtraceRegs).PC redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:130
./syscall_linux_arm64.go:113: (*PtraceRegs).SetPC redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:132
./syscall_linux_arm64.go:115: (*Iovec).SetLen redeclared in this block
        previous declaration at ./syscall_linux_amd64.go:134
./syscall_linux_arm64.go:115: too many errors
```

**Comment by bradfitz** [comment]
@minux is on it.

**Comment by davecheney** [comment]
Closed in 8bcc2dbee45
